doc/content/getting-started/aws/ecs/interop/_index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Interoperability"
+description: "LoRaWAN Backend Interfaces"
+weight: 3
+---
+
+{{% tts %}} exposes the Join Server API defined by LoRaWAN Backend Interfaces 1.0 and 1.1. This is used for interoperability between LoRaWAN networks. The functionality can be exposed either by the Join Server or by the Identity Server, but not both at once.
+
+Join Server is capable of handling the join flow, that is `HomeNSReq`, `JoinReq` and `AppSKeyReq` requests. Identity Server can only answer to `HomeNSReq`. Typically, you should enable interoperability if you want to use device activations through Packet Broker. You can use Join Server interoperability if you want to expose the entire join flow to other LoRaWAN network servers.
+
+# Join Server interoperability
+
+## Deployment
+
+### Step 1: deploy {{% tts %}}
+
+Deploy {{% tts %}} normally. For the `PeerRequesterAccountId` parameter in `1-1-vpc`, use the ID of the account where you want to deploy Crypto Server. Do not enable interop and assume that {{% tts %}} doesn't connect to the Crypto Server.
```

Comment:
There is no need to document anything about Crypto Server at this point. It is part of TTSE but it is unusable by customers in its current shape and form.

Reply:
I think we should document this. This documentation isn't only for the customers, but also for ourselves. I mean, you spent hours looking up how Global Join Server actually works, so when we're doing a new setup, it's worth it to document it, so we don't run into the same problem again in the future.

Also, without the Crypto Server part, the documentation boils down to "TTS supports interop", so I can simply close this PR without merging.

Reply:
OK